src/components/manage/LinksToItem/LinksToItem.jsx

```diff
@@ -30,134 +24,129 @@ const messages = defineMessages({
 
   useEffect(() => {
-    if (!content_subrequest[pathname]) {
-      dispatch(getContent(getBaseUrl(pathname), null, pathname));
-    }
-    return () => {
-      resetContent(pathname);
-    };
-  }, [dispatch, pathname, content_subrequest]);
+    dispatch(queryRelations(null, false, itempath, null, [itempath]));
+  }, [dispatch, itempath]);
 
   useEffect(() => {
-    if (
-      !relations_subrequest[pathname] &&
-      content_subrequest[pathname]?.data?.UID
-    ) {
-      dispatch(
-        queryRelations(null, false, pathname, null, [
-          content_subrequest[pathname]?.data?.UID,
-        ]),
-      );
-    }
-    return () => {
-      resetRelations(pathname);
-    };
-  }, [dispatch, content_subrequest[pathname]?.data?.UID, relations_subrequest]);
+    if (!title) dispatch(getContent(itempath));
+  }, [dispatch, itempath, title]);
 
   let links = {};
-
-  // Create a list of links (via constructing a hashmap and thus avoiding duplicates)
-  if (relations) {
-    for (const relation_items of Object.values(relations.items)) {
-      for (const item of relation_items) {
-        links[item.source.UID] = item.source;
-      }
-    }
+  if (myrelations) {
+    Object.keys(myrelations).forEach((relationtype) => {
+      links[relationtype] = {};
+      myrelations[relationtype].items.forEach((item) => {
+        links[relationtype][item.source.UID] = item.source;
+      });
+    });
   }
 
-  let links_ordered = Object.values(links).sort((link) => link['@id']);
-  const relations_found = links_ordered.length > 0;
-  return content && relations ? (
-    <Container id="linktoitem">
-      <Helmet title={intl.formatMessage(messages.linktoitem)} />
+  let links_ordered = {};
+  Object.keys(links).forEach((relationtype) => {
+    links_ordered[relationtype] = Object.values(links[relationtype]).sort(
+      (link) => link['@id'],
+    );
+  });
+
+  const relations_found = Object.keys(links_ordered).length > 0;
+  return (
+    <Container id="linkstoitem">
+      <Helmet title={intl.formatMessage(messages.linkstoitem)} />
       <Segment.Group raised>
         <Segment className="primary">
           <FormattedMessage
             id="Links to {title}"
             defaultMessage="Links to {title}"
-            values={{ title: <q>{content.title}</q> }}
+            values={{ title: <q>{title}</q> }}
           />
         </Segment>
-        {relations_found && (
+        {relations_found ? (
           <>
             <Segment secondary>
               <FormattedMessage
-                id="Whenever this item is being referenced from some different item by a hyperlink, block or similar, it appears here in this list."
-                defaultMessage="Whenever this item is being referenced from some different item by a hyperlink, block or similar, it appears here in this list."
+                id="Content that link to this item"
+                defaultMessage="Content that link to this item"
               />
             </Segment>
```

Comment:
I think this is redundant to the heading above it in the table and should be removed.